compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -1183,23 +1183,17 @@ impl VariantDef {
     ///
     /// If someone speeds up attribute loading to not be a performance concern, they can
     /// remove this hack and use the constructor `DefId` everywhere.
+    #[instrument(level = "debug", skip(recover_tainted, is_field_list_non_exhaustive))]
```

Comment:
why skip these :thinking: i think they are useful and feel like they've probalb yjust been added to this fn without updating the debug statement

Reply:
Force-pushed fix, as 3rd commit will be overkill.